backend/src/main/java/backend/security/SecurityAdapter.java

```diff
@@ -28,7 +28,8 @@
             "/api/images/{imageId}",
             "/api/users/{username}",
             "/api/password/reset",
-            "/api/tests"
+            "/api/tests",
+            "/api/utils/**"
```

Comment:
Nie trzeba wykluczać tego adresu z security

Reply:
to tylko do testowania w postmanie sobie wrzuciłem, przed mergem usunę ;)